docs/messaging/notifications.mdx

```diff
@@ -190,7 +190,7 @@ const AndroidNotificationChannel channel = AndroidNotificationChannel(
   'high_importance_channel', // id
   'High Importance Notifications', // title
   'This channel is used for important notifications.', // description
-  importance: Importance.max,
+  importance: Importance.Max,
```

Comment:
Looking at the source code this is already correct as lowercase `max`?

![image](https://user-images.githubusercontent.com/5347038/98967362-4dc84700-2504-11eb-80af-d8b4133a126a.png)

Reply:
My bad. Apparently my current project was using older version of `flutter_local_notification` and was still using the old enums.